core-service/src/test/java/com/crmfoodestablishment/coreservice/service/OrderServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package com.crmfoodestablishment.coreservice.service;
+
+import com.crmfoodestablishment.coreservice.dto.order.NewOrderDto;
+import com.crmfoodestablishment.coreservice.entity.Order;
+import com.crmfoodestablishment.coreservice.repository.DishRepository;
+import com.crmfoodestablishment.coreservice.repository.OrderRepository;
+import com.crmfoodestablishment.coreservice.service.exception.NotFoundException;
+import com.crmfoodestablishment.coreservice.service.impl.OrderServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+public class OrderServiceImplTest {
+
+    private OrderServiceImpl orderService;
+
+    @Mock
+    private OrderRepository orderRepository;
+
+    @Mock
+    private DishRepository dishRepository;
+
+    @Mock
+    private OrderMapper orderMapper;
+
+    @BeforeEach
+    public void init() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
why do u need this line? If I'm not mistaken with latest versions of JUnit and Mockito you can use only annotation @InjectMocks

Reply:
Yes, used @Injectmocks)